src/RetryAnnotationTrait.php

```diff
@@ -2,17 +2,20 @@
 
 namespace PHPUnitRetry;
 
-use LogicException;
+use InvalidArgumentException;
+use function array_slice;
+use function explode;
+use function is_callable;
+use function is_numeric;
+use function sprintf;
+use function var_export;
```

Comment:
regarding the import of symbols, the following PHP code works fine:

```php
namespace Acme;

var_dump('Hello');
```

However this is thanks to an ugly PHP hack where PHP internally will try `Acme\var_dump()` and then fallback to `var_dump()` which have 3 consequences:

- It is slower (it would however be a micro-optimization especially here although some projects can see as much as a 15% speed gain from it)
- It is relying on a fallback: why not providing explicitly the dependency here like for classes?
- Makes it harder for SA tools

Whereas having the `use` statement has the only drawback of adding more use statements...

Reply:
Also, if someone adds library which defines a `array_slice` function in the namespace `PHPUnitRetry`, it will override the behavior in my plugin, which would not be cool.